lib/testutils/rules.go

```diff
@@ -20,82 +20,81 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
-var ipv4 = 4
-var ipv6 = 6
-var strProtocol1 = numorstring.ProtocolFromString("icmp")
-var strProtocol2 = numorstring.ProtocolFromString("udp")
-var numProtocol1 = numorstring.ProtocolFromInt(240)
-
-var portRange, singlePort, namedPort numorstring.Port
+var InRule1, InRule2, EgressRule1, EgressRule2 apiv3.Rule
 
 func init() {
-	var err error
-	portRange, err = numorstring.PortFromRange(10, 20)
+	portRange, err := numorstring.PortFromRange(10, 20)
 	if err != nil {
 		logrus.WithError(err).Panic("Failed to create port range")
 	}
-	singlePort = numorstring.SinglePort(1024)
-	namedPort = numorstring.NamedPort("named-port")
-}
+	singlePort := numorstring.SinglePort(1024)
+	namedPort := numorstring.NamedPort("named-port")
 
-var icmpType1 = 100
-var icmpCode1 = 200
+	ipv4 := 4
+	ipv6 := 6
+	strProtocol1 := numorstring.ProtocolFromString("TCP")
```

Comment:
We do this a lot in the Felix UTs/FVs too so either need to be permissive or fix those up too.